openapi.yaml

```diff
@@ -16,175 +16,511 @@ host: api.oregonstate.edu
 basePath: /v1
 externalDocs:
   description: GitHub Repository
-  url: https://github.com/osu-mist/express-api-skeleton
+  url: https://github.com/osu-mist/poker-api
 produces:
   - application/json
 paths:
-  /pets:
-    get:
-      summary: List all pets
+  /games:
+    post:
+      summary: Create a poker game
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+        - games
+      consumes:
+        - application/json
+      operationId: postGames
       parameters:
+        - in: body
+          name: requestBody
+          schema:
+            $ref: '#/definitions/GamePostBody'
+      responses:
+        '201':
+          description: Successfully update a game
+          schema:
+            $ref: '#/definitions/GameResult'
+          headers:
+            Location:
+              type: string
+              format: url
+              description: Location of the newly created game
+        '400':
+          description: Invalid game object
+          schema:
+            $ref: '#/definitions/Errors'
+        '404':
+          description: No such a game
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+  /games/{gameID}:
+    get:
+      summary: Info for a specific game
+      tags:
+        - games
+      description: get game by game id
+      operationId: getGameByID
+      parameters: 
         - $ref: '#/parameters/authorization'
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
-        - in: query
-          name: species
-          description: Search by pet species
-          required: false
-          type: string
+        - name: gameID
+          in: path
+          description: unique integer
+          required: true
+          type: integer
       responses:
         '200':
           description: Successful response
           schema:
-            $ref: '#/definitions/PetsResult'
+            $ref: '#/definitions/GameResult'
+        '404':
+          description: Game not found
+          schema:
+            $ref: '#/definitions/Errors'
+        '500':
+          description: Internal server error
+          schema:
+            $ref: '#/definitions/Errors'
+    put:
+      tags:
+        - games
+      summary: Update a game specified by its id
+      operationId: updateGameByID
```

Comment:
I think we should use the same HTTP verbs as you did for `operationId: postGames` for the consistency. Same comments with your other methods.